src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -71,9 +72,9 @@ internal sealed class TextBufferIssueTracker : IIssueTracker, ITagger<IErrorTag>
             this.logger = logger;
 
             this.document = document;
-            this.LastAnalysisFilePath = document.FilePath;
-
-            this.Factory = new IssuesSnapshotFactory(LastAnalysisFilePath);
+            LastAnalysisFilePath = document.FilePath;
+            fileTracker.AddFiles(LastAnalysisFilePath);
```

Comment:
Can we also update the file encoding here? Currently on ListFiles we always use the UTF8 default. It is passed to the SLCore analyzer for legacy reasons, but it can't be passed to the `analyzeFilesAndTrack` method of sloop. We might update the encoding here instead and later, when we move all languages to sloop, remove it from the analyzer interface

Reply:
If it would take a considerable amount of time to do this, then just create a ticket

Reply:
https://sonarsource.atlassian.net/browse/SLVS-1219